crates/oxc_ecmascript/src/constant_evaluation/mod.rs

```diff
@@ -15,6 +17,8 @@ pub use value::ConstantValue;
 pub use value_type::ValueType;
 
 pub trait ConstantEvaluation<'a>: MayHaveSideEffects {
+    fn create_expression_numeric(&self, value: f64) -> Expression<'a>;
+
```

Comment:
It feels wrong to add this here. But I didn't come up with other ideas...

Reply:
How about adding a getter for `AstBuilder`?